client/src/components/QueryComponent/QueryComponent.js

```diff
@@ -62,7 +76,30 @@ class QueryComponent extends Component {
   }
 
   onChange = evt => {
+    let count = 0;
     this.setState({ selectedQuery: evt.target.value, selectedQueryKey: null });
+    evt.target.value.split("").map(character => {
```

Comment:
Maybe something like `const count = evt.target.value.split("").filter(c => c === "\n").length` might be better? I don't think map is quite right here as we don't use the output.